spec/feature/authenticated_user_spec.rb

```diff
@@ -91,14 +91,14 @@
 
```

Comment:
Glad to see the tests being changed. Excited to get them all back in working order.